utils/progressbar/progressbarmng.go

```diff
@@ -194,11 +194,25 @@ func (bm *ProgressBarMng) NewStringProgressBar(headline string, updateFn func()
 		mpb.BarRemoveOnComplete(),
 		mpb.PrependDecorators(
 			decor.Name(headline),
-			decor.Any(func(statistics decor.Statistics) string {
-				return updateFn()
+		),
+	)
+	return pb
+}
+
+func (bm *ProgressBarMng) NewDoubleValuesProgressBar(headLine string, color Color, totalTasks int64, updateFunc func() string) *TasksProgressBar {
```

Comment:
This function receives only one headline as an argument. The other one ("Files") is hard coded inside the function, and isn't received as an argument, which make the usage a bit strange.
Let's have have the first two arguments be `headLine1,  headLine2 string`

This function isn't really generic. Meaning, it doesn't really allow you to create any type of "double value progress" you wish. It is very specific to our use case, which is showing the total of files and total storage. For example, the function specifically uses `"%.1f/%.1f"`, which is good for our use case only.
I see two options to design this properly - 
1. Either make the function truely generic, allowing the user to configure it to show any two values.
2. Keep it specific for our use case, but name it "NewTotalFilesAndStorageProgressBar". Also In this case, it doesn't make sense that the update logic will be separated into an `updateFunc()` that is sent as an argument, becuase this progress cannot be customized anyway.

Reply:
1. I updated the input to two headlines
2. all the progress bars c'tors used a specific updating format in the body of the c'tor, I also think that is more generic and elegant to make the showing update format more generic but then I think that we have to change all the progress bars c'tors.. what you think Eyal? 
3. the updateFunc() is not an option, it's one of the conditions to use (Any) decorate in the bar library so we just added it

Reply:
It is very generic now